sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -1070,26 +1070,24 @@ def hpat_pandas_stringmethods_isdecimal_impl(self):
 }
 
 
-@sdc_overload_method(StringMethodsType, 'capitalize')
-def hpat_pandas_stringmethods_capitalize(self):
-    """
+sdc_pandas_series_str_docstring_template_ = """
```

Comment:
I vote for using common doctring template and inserting block seealso to this template where all the blocks seealso can be created once and used in the rendering of docstring for each method. Something like that:

```python
sdc_pandas_series_str_docstring_tmpl = """
<...>
    .. seealso::
        {seealso}
<...>
"""

seealso_check_methods = """
    :ref:`Series.str.is<...> <pandas.Series.str.is<...>
    <...>
"""

seealso_transform_methods = """
    :ref:`Series.str.<other_method> <pandas.Series.str.<other_method>
    <...>
"""

stringmethods_funcs = {
    <method>: {'method': <...>, 'caption': <...>, 'seealso': <seealso_check_methods or seealso_transform_methods>},
```

Please do visual check of the documentation after applying the changes.